### ISSUE TITLE
Utils should be structured into Struct of Classes

### ISSUE THREAD

**Issue by olivierobert** [tiendnuit/Nimbl3Test#2]
The methods in `Utils` are helpers methods, extensions... a bunch of unrelated methods that have nothing to do with each other. It's best practice to group these utils into named struct or classes e.g. `AlertUtils`, `ViewUtils`...

**Comment by tiendnuit** [comment]
Yes, I see, thanks for your advance. Actually, in my real project, I separated it into small file with related methods.
